File: jdroid-java-webapp-core/src/test/java/com/jdroid/javaweb/commons/AbstractTest.java
package com.jdroid.javaweb.commons;

import com.jdroid.javaweb.application.TestApplication;

import org.testng.annotations.BeforeClass;

public class AbstractTest {
	
	@BeforeClass
	public void onBeforeClass() {
		TestApplication testApplication = new TestApplication();
		testApplication.init();
	}
}

File: jdroid-java-webapp-core/src/test/java/com/jdroid/javaweb/commons/AbstractTest.kt
package com.jdroid.javaweb.commons

import com.jdroid.javaweb.application.TestApplication
import org.junit.BeforeClass

open class AbstractTest {

    companion object {

        @JvmStatic
        @BeforeClass
        fun onBeforeClass() {
            val testApplication = TestApplication()
            testApplication.init()
        }
    }
}
